test/features/step_definitions/sdk_steps.rb

```diff
@@ -17,3 +18,20 @@
     When I click on #{action_button} ()
   }
 end
+
+element = '(\w+(?:|\(.*\)) \(\w*\)(?:| index \d+))'
+
+When(/^I open #{element} in a new tab$/) do |element|
+  url = element.text
```

Comment:
if the element is textarea the url can be retrieved with `element.value`, that way we respect the standard.

Reply:
I linked an old version of the docs. In the version we are using `element.value` has been removed so we must use `element.attribute('value')`